src/main/kotlin/no/nav/syfo/cronjob/DialogmoteCronjobRunner.kt

```diff
@@ -0,0 +1,56 @@
+package no.nav.syfo.cronjob
+
+import kotlinx.coroutines.coroutineScope
+import kotlinx.coroutines.delay
+import kotlinx.coroutines.launch
+import net.logstash.logback.argument.StructuredArguments
+import no.nav.syfo.application.ApplicationState
+import no.nav.syfo.cronjob.leaderelection.LeaderPodClient
+import org.slf4j.LoggerFactory
+import java.time.Duration
+
+class DialogmoteCronjobRunner(
+    private val applicationState: ApplicationState,
+    private val leaderPodClient: LeaderPodClient,
```

Comment:
Hadde koden blitt et lite hakk enklere dersom cronjob hadde vært en del av tilstanden her (i stedet for parametre til start og run-metodene)? Dessuten åpner koden nedenfor at man kjøre start og run med ulik parameter.